README.md

```diff
@@ -2,12 +2,12 @@
 
 ## Client
 
-Run `cd client && npm install && npm start`
+Run `cd client && yarn start`
```

Comment:
You remove the `npm install` command.
You still need to run `yarn`

Reply:
This is back. Thank you